.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -5,7 +5,8 @@
 <!-- Describe your changes below in as much detail as possible -->
 
 #### Scope
-
-- [ ] Patch: Bug Fix
-- [ ] Minor: New Feature
-- [ ] Major: Breaking Change
+<!-- Pick one:
+Patch: Bug Fix
+Minor: New Feature
+Major: Breaking Change
```

Comment:
```suggestion
```

Reply:
we might as well remove this, as we do not except breaking changes from contributors

Reply:
I have committed the change although I think contributors should still be allowed to send PRs that have a breaking change (like a bug fix could be a breaking change), you then take a call whether to merge it when you release a major or reject it altogether.

Reply:
hm, that's a good point 🤔 I'd be happy to merge either way, let me know what you perefer

Reply:
I guess we should keep as it was, I'll revert in that case